crates/re_query/src/range/results.rs

```diff
@@ -747,9 +761,10 @@ impl RangeComponentResultsInner {
             }),
             "back promises must always be sorted in ascending index order"
         );
-        if let (Some(p_index), Some(i_index)) =
-            (promises_back.last().map(|(index, _)| index), indices.back())
-        {
+        if let (Some(p_index), Some(i_index)) = (
+            promises_back.first().map(|(index, _)| index),
```

Comment:
Changing this this didn't catch any new sanity issues, but I think this was just a typo unless I'm misunderstanding the intent.

Reply:
Yep, nice catch